templates/about.html

```diff
@@ -1,113 +1,178 @@
             <a href="https://github.com/PeterDaveHello" target="_blank">
               <img style="width: 50px; height: 50px;" alt="Peter Dave Hello" src="/img/PeterDaveHello.jpg" />
               Peter Dave Hello
             </a>
-            <a style="margin-top: 10px;" class="twitter-follow-button" href="https://twitter.com/peterdavehello" data-show-count="false" data-lang="en">Follow @peterdavehello</a>
+            <a style="margin-top: 10px;" class="twitter-follow-button" href="https://twitter.com/peterdavehello"
+               data-show-count="false" data-lang="en">Follow @peterdavehello</a>
           </td>
           <td style="width: 300px;">
             <a href="http://decompile.it/blog/" target="_blank">
               <img style="width: 50px; height: 50px;" alt="Drew Freyling" src="/img/drew.jpg" />
               Drew Freyling
             </a>
-            <a style="margin-top: 10px;" class="twitter-follow-button" href="https://twitter.com/drewfreyling" data-show-count="false" data-lang="en">Follow @drewfreyling</a>
+            <a style="margin-top: 10px;" class="twitter-follow-button" href="https://twitter.com/drewfreyling"
+               data-show-count="false" data-lang="en">Follow @drewfreyling</a>
           </td>
           <td style="width: 300px;">
             <a href="https://petecooper.org" target="_blank">
               <img style="width: 50px; height: 50px;" alt="Pete Cooper" src="/img/pete.jpg" />
               Pete Cooper
             </a>
-            <a style="margin-top: 10px;" class="twitter-follow-button" href="https://twitter.com/petecooper" data-show-count="false" data-lang="en">Follow @petecooper</a>
+            <a style="margin-top: 10px;" class="twitter-follow-button" href="https://twitter.com/petecooper"
+               data-show-count="false" data-lang="en">Follow @petecooper</a>
           </td>
         </tr>
-      </tbody>
-    </table>
-  </div>
+        </tbody>
+
+        <thead>
+        <tr>
+          <td><h3>Community</h3></td>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <td style="width: 300px;">
+            <a href="https://mattcowley.co.uk" target="_blank">
+              <img style="width: 50px; height: 50px;" alt="Matt Cowley" src="/img/matt.jpg" />
+              Matt Cowley
+            </a>
+            <a style="margin-top: 10px;" class="twitter-follow-button" href="https://twitter.com/MattIPv4"
+               data-show-count="false" data-lang="en">Follow @MattIPv4</a>
+          </td>
+        </tr>
+        </tbody>
+      </table>
+    </div>
   </div>
 </div>
 
-    <script async>window.twttr=(function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],t=window.twttr||{};if(d.getElementById(id))return t;js=d.createElement(s);js.id=id;js.src="https://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);t._e=[];t.ready=function(f){t._e.push(f);};return t;}(document,"script","twitter-wjs"));</script>
```

Comment:
I wonder if we need to touch this one as we won't even edit it?

Reply:
Ah yes, sorry, hadn't noticed that the refactor had done that. Will rectify when I'm home later!